plugins/modules/cloud/lxd/lxd_project.py

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: lxd_project
+short_description: Manage LXD projects
+description:
+  - Management of LXD projects
+author: "Raymond Chang (@we10710aa)"
+options:
+    name:
+        description:
+          - Name of a project.
+        required: true
+        type: str
+    description:
+        description:
+          - Description of the project.
+        type: str
+    config:
+        description:
+          - 'The config for the project (e.g. {"features.profiles": "true"}).
+            See U(https://linuxcontainers.org/lxd/docs/master/projects/)'
```

Comment:
Paragraphs should always contain complete sentences.
```suggestion
            See U(https://linuxcontainers.org/lxd/docs/master/projects/).'
```